shell/platform/linux/fl_basic_message_channel.cc

```diff
@@ -232,7 +232,6 @@ G_MODULE_EXPORT void fl_basic_message_channel_send(FlBasicMessageChannel* self,
                                                    GAsyncReadyCallback callback,
                                                    gpointer user_data) {
   g_return_if_fail(FL_IS_BASIC_MESSAGE_CHANNEL(self));
-  g_return_if_fail(message != nullptr);
```

Comment:
This will mean calling `fl_message_codec_encode_message` with null below; does that work? Is there unit testing of it? Does it return a non-null data pointer, such that line 246 won't return an error?

Reply:
Please add '(allow-none)' to the documentation string in `public/flutter_linux/fl_basic_message_channel.h`

Reply:
> Please add '(allow-none)' to the documentation string in public/flutter_linux/fl_basic_message_channel.h

Done, thanks for noticing this :+1: